tsconfig.json

```diff
@@ -0,0 +1,6 @@
+//TypeScript is configured using tsconfig.json. 
+//Even if you don’t write TypeScript code, this file
+//is important so that tools like Astro and VS Code know how to understand your project.
```

Comment:
I think this kind of commentary can be left out (:

Reply:
Sorry, yes the PR was rather large because of a lot of just setting up the file hierarchy with duplicate files. The next PRs will be a lot smaller. Is there a specific range of PR size that I should aim for?
- Removed all the DS_Store files and added them to .gitignore
- Yep, a lot of the files in src are just duplicates of the initial static website. I kept the originals just in case I messed something up and we needed to revert or deploy the original project. I can delete them all now if you would like?
- Added the empty line at the end of all html and astro files :D

Reply:
Usually you want to keep your PR to less than a 1000 lines (ideally 500).

As for the duplicate files, it's always recommended to delete them as you can always go back in the history to retrieve anything you need. Duplicating files will also cause git to see them as new rather than seeing them as moved which would have made for a much cleaner PR.

I hope that you are using a linter to add the new lines at the end of files (something that should be added through a npm script) so that you are not doing it manually and that can be enforced during the CI/CD process.